backend/entities/Models/ParcelFilter.cs

```diff
@@ -11,6 +11,16 @@ namespace Pims.Dal.Entities.Models
     public class ParcelFilter
     {
         #region Properties
+        /// <summary>
+        /// get/set - The page number.
+        /// </summary>
+        public int Page { get; set; } = 1;
+
+        /// <summary>
+        /// get/set - The quantity of parcels to return in a single request.
+        /// </summary>
+        public int Quantity { get; set; } = 10;
+
```

Comment:
Ideally I'd like to see this in a base class or interface or there will be lot's of copy paste (and associated manual errors) if/when we start supporting paginating other resources/entities in the system - `api/users` comes to mind (not 100% sure what the name is for the users endpoint but we will need to paginate it at some point). 
I'm sure we will run into other large lists of entities as PIMS grows - thoughts?

Reply:
The next PR I'm working on does this.